src/firefly/plans/regions_display.py

```diff
@@ -164,14 +164,11 @@ def update_total_time(self):
         self.ui.label_sec_total.setText(str(secs_total))
 
     def time_calculate_method(self, detector_time):
-        num_points = self.ui.scan_pts_spin_box.value()
-        total_time_per_scan = detector_time * num_points
-        return total_time_per_scan
+        pass
```

Comment:
I think this should be something other than ``pass``.

Right now, it will return ``None``. If that's what you want, then explicitly ``return None``.

If you expect every subclass to define its own version of this method, then use ``raise NotImplementedError`` so that if a subclass doesn't do that, an obvious exception is thrown. Otherwise you'll get `TypeError`s in different parts of your code that will be hard to track down.